### ISSUE TITLE
Camouflage doesn't work with Chisel connected textures

### ISSUE THREAD

**Issue by desht** [TeamPneumatic/pnc-repressurized#74]
Using connected-texture chisel blocks as camouflage doesn't connect the textures.

We need to at least implement ``IFacade`` from the CTM API, and add this to ``BlockPneumaticCraftCamo``:
```java
    @Nonnull
    @Override
    @Optional.Method(modid = "ctm-api")
    public IBlockState getFacade(@Nonnull IBlockAccess world, @Nonnull BlockPos pos, @Nullable EnumFacing side) {
        TileEntity te = getTE(world, pos);
        if (te instanceof ICamouflageableTE && ((ICamouflageableTE) te).getCamouflage() != null) {
            return ((ICamouflageableTE) te).getCamouflage();
        }
        return world.getBlockState(pos);
    }
```

This allows chisel blocks to see our camo blocks as chisel blocks, and use connected textures.  But our own camouflage does not connect, just using the unconnected texture version.

**Comment by desht** [comment]
https://github.com/TeamPneumatic/pnc-repressurized/commit/5cd856dc2c87f735d3440f7c9e1be03b8c025dfa

**Comment by desht** [comment]
Fixed in 0.8.1
